src/redux/slices/DoctorSlice/index.ts

```diff
@@ -0,0 +1,40 @@
+import cookie from 'utils/functions/cookies';
+import { createSlice } from '@reduxjs/toolkit';
+
+import { persistor } from '@redux/store';
+import DoctorInitialState from './types';
+
+const initialState: DoctorInitialState = {
+  address: undefined,
+  birthDate: undefined,
+  city: undefined,
+  country: undefined,
+  email: undefined,
+  firstName: undefined,
+  gender: undefined,
+  id: undefined,
+  isVerified: undefined,
+  lastName: undefined,
+  phoneNumber: undefined,
+  photoUrl: undefined,
+  role: undefined,
+  specialityId: undefined,
+  timeZone: undefined,
+};
```

Comment:
can you make init state`{}`?, this code look weard

Reply:
fixed